packages/core/src/lib/TableNew/Helper.tsx

```diff
@@ -0,0 +1,80 @@
+export function paginate(
```

Comment:
Please use the <Pagination/> component added to the kit.
It becomes difficult to maintain two separate components better to use the existing one, and add features to it if anything is needed 😄

Reply:
I didn't know, that there is a pagination component 🙈